cpp/tests/CMakeLists.txt

```diff
@@ -741,6 +741,7 @@ ConfigureTest(
   streams/text/ngrams_test.cpp
   streams/text/replace_test.cpp
   streams/text/stemmer_test.cpp
+  streams/text/subword_tokenize_test.cpp
```

Comment:
Does this file exist? I think you might have forgotten to add it to the PR?

Reply:
omg, I was wondering how it builds on my machine lol
Added now!